.vitepress/theme/scripts/gtag.ts

```diff
@@ -0,0 +1,21 @@
+declare global {
+  interface Window {
+    gtag?: (eventName: string, options: Record<string, unknown>) => void;
+  }
+}
+
+// https://developers.google.com/tag-platform/gtagjs/reference
+interface Event {
+  eventName: string;
+  pagePath: string;
+  value: string;
+}
+
+export function sendEvent({ eventName, pagePath, value }: Event) {
```

Comment:
❓ I can at least see the events in GA Realtime overview when I click 👍/👎 buttons so it seems to be working. Could someone check if I set it up correctly?